acceptance/tests/modules/build/build_agent.rb

```diff
@@ -1,5 +1,8 @@
 test_name "puppet module build (agent)"
 
+tag 'audit:medium',
+    'audit:acceptance'
```

Comment:
The method with which Puppet creates a tarball is OS specific. I'm still not sure if this shouldn't be an integration test (does not care about packaging).